resources/js/components/FormConditions.vue

```diff
@@ -19,6 +29,28 @@ export default {
         if (csrfField && token && token !== 'STATAMIC_CSRF_TOKEN') {
             csrfField.value = token
         }
-    }
+    },
+
+    methods: {
+        async submit(event) {
+            event.preventDefault()
+
+            const response = await rapidezFetch(this.$el.action, {
+                method: this.$el.method,
+                body: new FormData(this.$el),
+            })
+
+            if (response.ok) {
+                this.success = true
+                Notify(window.config.translations.packages.statamic.form.success, 'success')
+                if (this.callback) {
+                    await this.callback()
+                }
+            } else {
+                this.error = true
+                Notify(window.config.translations.packages.statamic.form.error, 'error')
```

Comment:
In some places in Rapidez we also log the error https://github.com/rapidez/core/blob/master/resources/js/fetch.js#L150
https://github.com/rapidez/core/blob/4ddd4d2c1fd0a3f6ae0f2dea81331a0a5ef7254a/resources/js/stores/useSwatches.js#L21
is that something we might want to do here?